include/NovelRT/Timing/StepTimer.h

```diff
@@ -5,6 +5,7 @@
 // Original code is available under the MIT Licence
 
 #include "Timestamp.h"
+#include "../Utilities/Event.h"
```

Comment:
What does this do?

Reply:
`StepTimer.h` depends on `Event.h`. Without this, including `StepTimer.h` alone doesn't work.

Reply:
...how on earth was this working previously? Can you figure that out?

Reply:
It worked only with of the main `NovelRT.h` header, since it included headers in the right order (Event first, Timer second): https://github.com/novelrt/NovelRT/blob/main/include/NovelRT/NovelRT.h#L103

Reply:
I see. Thanks.